posthog/session_recordings/models/session_recording.py

```diff
@@ -245,7 +190,7 @@ def set_start_url_from_urls(self, urls: Optional[List[str]] = None, first_url: O
         self.start_url = url.split("?")[0][:512] if url else None
 
 
-@receiver(models.signals.post_save, sender=SessionRecording)
+@mutable_receiver(models.signals.post_save, sender=SessionRecording)
```

Comment:
What's this?

Reply:
It lets us turn off the receiver in tests. 

Otherwise every time you save a session recording the on_save logic kicks in and the tests get complicated

Reply:
Oooh cool